src/main/java/org/apache/maven/plugins/dependency/analyze/AbstractAnalyzeMojo.java

```diff
@@ -228,11 +228,15 @@ public abstract class AbstractAnalyzeMojo extends AbstractMojo {
      * For example, <code>org.apache.*</code> will match all artifacts whose group id starts with
      * <code>org.apache.</code>, and <code>:::*-SNAPSHOT</code> will match all snapshot artifacts.
      * </p>
+     * <p>
+     * By default, org.slf4j:slf4j-simple is ignored. Setting this property to an empty list
+     * will allow it to be detected.
+     * </p>
      *
      * @since 2.10
      */
     @Parameter
-    private String[] ignoredUnusedDeclaredDependencies = new String[0];
+    private String[] ignoredUnusedDeclaredDependencies = {"org.slf4j:slf4j-simple::"};
```

Comment:
1. why not use a `default` option of parameter?
2. when user defined another ignores list it will be overridden ... maybe we need add it latter to be always ignored ...

Reply:
A little later in the file I see this comment

```
    // defaultValue value on @Parameter - not work with Maven 3.2.5
    // When is set defaultValue always win, and there is no possibility to override by plugin configuration.
```